cypress/e2e/marc/marc-authority/reporting-marc-authority/marc-authority-reporting-not-include-data-deleted-record.cy.js

```diff
@@ -120,23 +121,22 @@ describe('MARC', () => {
           });
 
           cy.loginAsAdmin();
```

Comment:
In this case it's better to use something like this

```
 cy.loginAsAdmin({
            path: TopMenu.inventoryPath,
            waiter: InventoryInstances.waitContentLoading,
          }).then(() => {
            InventoryInstances.searchByTitle(testData.createdAuthorityID[0]);
            InventoryInstances.selectInstance();
            InventoryInstance.editMarcBibliographicRecord();
           .....

```

Because you can't avoid very first time visit call, so you can navigate to nessesary page within admin log in call.

Reply:
Done